File: src/main/java/pl/arturczopek/mycoach/repository/ExerciseSessionRepository.java
package pl.arturczopek.mycoach.repository;

import org.springframework.data.repository.CrudRepository;
import pl.arturczopek.mycoach.model.database.ExerciseSession;

/**
 * @Author Artur Czopek
 * @Date 06-11-2016
 */
public interface ExerciseSessionRepository extends CrudRepository<ExerciseSession, Long> {
}

File: src/main/kotlin/pl/arturczopek/mycoach/repository/ExerciseSessionRepository.kt
package pl.arturczopek.mycoach.repository

import org.springframework.data.repository.CrudRepository
import pl.arturczopek.mycoach.model.database.ExerciseSession

/**
 * @Author Artur Czopek
 * @Date 06-11-2016
 */

interface ExerciseSessionRepository : CrudRepository<ExerciseSession, Long>
